### PR TITLE
SSB: Bugfixes and Cosmetic Updates

### PR DESCRIPTION
**Pull request by xInstruct**
TODO:
Averardo -> Lunala

SectoniaServant -> Sectonia

other stuff i might have forgotten

### CODE REVIEW

File: data/mods/ssb/conditions.ts [hunk 1]
```diff
@@ -699,7 +699,7 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			this.add(`c|${getName('grimAuxiliatrix')}|${['THE JUICE IS LOOSE', 'TOOTHPASTE\'S OUT OF THE TUBE', 'PREPARE TO DISCORPORATE'][this.random(3)]}`);
 		},
 		onFaint() {
-			this.add(`c|${getName('grimAuxiliatrix')}|${['NOT LIKE THIS', 'HALT - MODULE CORE HEMORRHAGE', 'AAAAAAAAAAAAAAAAAAA'][this.random(3)]}`);
+			this.add(`c|${getName('grimAuxiliatrix')}|${['NOT LIKE THIS', 'HALT - MODULE CORE HEMORRHAGE', 'AAAAAAAAAAAAAAAAAAA', 'Change da world... my final message. Goodb ye.'][this.random(4)]}`);
```
Comment (AnnikaCodes) [inline data/mods/ssb/conditions.ts:702]:
```suggestion
			this.add(`c|${getName('grimAuxiliatrix')}|${this.sample(['NOT LIKE THIS', 'HALT - MODULE CORE HEMORRHAGE', 'AAAAAAAAAAAAAAAAAAA', 'Change da world... my final message. Goodb ye.'])}`);
```
This way, you don't have to worry about updating the length! (I'd also make the phrase an intermediate variable.)

Comment (xInstruct) [reply data/mods/ssb/conditions.ts:702]: i mean sure itll just be not consistent with how its done everywhere else

**Comment by xInstruct** [comment]
Don't merge yet thanks its 2am

**Comment by xInstruct** [comment]
Ok everything is tested and works properly:tm:

**Comment by Zarel** [comment]
Are we doing this as a batch or is this high enough priority to merge ASAP?

**Comment by xInstruct** [comment]
> Are we doing this as a batch or is this high enough priority to merge ASAP?

Was talking w/ Kris and Aelita earlier (RIP AFKHero) about the future of staff bros. After this PR, SSB4 will probably be more or less deprecated to an extent that balance patches will not happen much, and namechanges will be done sporadically.

**Comment by xInstruct** [comment]
ofc bugfixes will still happen time to time, but the format deserves a break since the next one will be a massive juggernaut.

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**